site/public/js/calendar.js

```diff
@@ -104,6 +109,67 @@ function generateCalendarItem(item) {
         </a>`;
 }
 
+function generateItemHtml(item, date) {
+    let styleString = '';
+    switch (item['type']) {
+        case 'announcement':
+            styleString = 'style="font-weight: bold;"';
```

Comment:
what happens if someone comes along later and decides elsewhere in the code that they want to put an inline style?  It might be better to use a CSS class to indicate this.  Bootstrap has the `.font-weight-bold` and `.font-italic` classes which might be helpful here.

Reply:
This code is going to be removed as I realized there already exists some code for styling items.